LICENSE

```diff
@@ -0,0 +1,16 @@
+Copyright (c) 2016
```

Comment:
`Copyright (c) 2016` => `Copyright (c) 2016, Open Pension Team and contributors.`